packages/debug-utils/index.js

```diff
@@ -530,13 +530,16 @@ var DebugUtils = {
     }
   },
 
-  formatInstruction: function (traceIndex, traceLength, instruction) {
+  formatCurrentInstruction: function (instruction) {
+    const pc = this.formatPC(instruction.pc);
+    const fmtInstr = this.formatInstruction(instruction);
```

Comment:
This, by contrast, is the sort of overly-abbreviated variable name I would warn against.  I'd suggest expanding this to `formattedInstruction`; `fmtInstr` isn't really clear.

Reply:
rename to formattedInstruction, thx